src/pages/generate/pages/result/components/GeneratedImgA.tsx

```
@@ -12,16 +11,12 @@ import { useABTest } from '@/pages/generate/hooks/useABTest';
-  logResultImgClickMoreModalMakeNew,
   logResultImgSwipeSlideLeft,
   logResultImgSwipeSlideRight,
 } from '@/pages/generate/utils/analytics';
-import { useMyPageUser } from '@/pages/mypage/hooks/useMypage';
-import type { MyPageUserData } from '@/pages/mypage/types/apis/MyPage';
-import { ROUTES } from '@/routes/paths.ts';
-import GeneralModal from '@/shared/components/overlay/modal/GeneralModal';
+import CommunityComingSoonModal from '@/shared/components/overlay/modal/CommunityComingSoonModal';
 
-import LockIcon from '@shared/assets/icons/lockIcon.svg?react';
+import generateResultLockedPreview from '@assets/images/generateResultLockedPreview.png';
```

ESLint

[error] `@assets/images/generateResultLockedPreview.png` import should occur before import of `overlay-kit`

(import/order)